osu.Game.Rulesets.Catch/UI/CatchTouchInputMapper.cs

```diff
@@ -35,6 +35,8 @@ public class CatchTouchInputMapper : VisibilityContainer
         private void load(CatchInputManager catchInputManager, OsuColour colours)
         {
             const float width = 0.15f;
+            // Ratio between normal move area height and total input height
+            const float normal_area_height_ratio = 0.45f;
```

Comment:
Wait, does this mean that normal movement is using the shorter boxes? Isn't that backwards? I'd expect the dash buttons to be shorter if anything, because they're used more sparingly...

If reachability of both buttons is the issue here, then they should arguably be the same height, but not extend to the full height of the screen - i.e. you'd leave a bit of a gap with no inputs at the top of the playfield.

Reply:
I'm putting the split position a bit lower than the screen's center to appeal to players holding the device from lower corner to support the device's weight with their pinkies. Putting it in the middle is a bit weird on phones and medium sized tablets, as its too far away to put your pinky fingers beneath the screen to hold. I tried to account for largest amount of devices possible before we get customization. However, this is my own decision, where to split the areas can totally be discussed.

However, I don't like the idea of cutting a bit of dash from the top. It's literally wasted space, and shouldn't cause harm.
We can push it under the health bar though, but that's too much work for something temporary like this.